docs/management/action-types.asciidoc

```diff
@@ -59,6 +59,10 @@ a| <<swimlane-action-type,{swimlane}>>
 
 | Create an incident in {swimlane}.
 
+a| <<tines-action-type,Tines>>
```

Comment:
@lcawl @mikecote Do we list all of the connectors regardless of where they are supported (stack, security solution, o11y, cases, etc)?

Reply:
Yes, we've been keeping this page as the source of truth for the connector documentation.